tests/support/mssql/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [ "$1" = '/opt/mssql/bin/sqlservr' ]; then
     # starting it again.
     function initialize_app_database() {
         timeout=20
-        limit="$((SECONDS + timeout))}"
+        limit="${((SECONDS + timeout))}"
```

Comment:
I think the original typo was the _addition_ of a curly brace, not the missing of one.